src/Serilog.Sinks.Console/Sinks/SystemConsole/Themes/SystemConsoleTheme.cs

```diff
@@ -40,20 +40,18 @@ public class SystemConsoleTheme : ConsoleTheme
         /// </summary>
         public static SystemConsoleTheme Colored { get; } = SystemConsoleThemes.Colored;
 
-        readonly IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> _styles;
-
         /// <summary>
         /// Construct a theme given a set of styles.
         /// </summary>
         /// <param name="styles">Styles to apply within the theme.</param>
         public SystemConsoleTheme(IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> styles)
         {
             if (styles == null) throw new ArgumentNullException(nameof(styles));
-            _styles = styles.ToDictionary(kv => kv.Key, kv => kv.Value);
+            Styles = styles.ToDictionary(kv => kv.Key, kv => kv.Value);
         }
 
         /// <inheritdoc/>
-        public IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> Styles => _styles;
+        public IReadOnlyDictionary<ConsoleThemeStyle, SystemConsoleThemeStyle> Styles { get; private set; }
```

Comment:
👍 - if assignment only happens in the constructor, this doesn't need `private set` - ` { get; }` alone will do the job.

Reply:
fixed